packages/data/addon/models/navi-dimension.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+import EmberObject from '@ember/object';
+import { DimensionColumn } from 'navi-data/adapters/dimensions/interface';
+
+export default class NaviDimensionModel extends EmberObject {
+  /**
+   * Dimension column definition
```

Comment:
`@property` jsdoc

Reply:
Do we need this? VSCode seems to understand this is a property:

<img width="501" alt="Screen Shot 2020-08-11 at 10 58 40 PM" src="https://user-images.githubusercontent.com/2983409/89973578-486a1080-dc26-11ea-8b6c-f3d9743d70d6.png">

Reply:
I think you're right